src/components/Calculator.js

```diff
@@ -1,35 +1,39 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them. By the way, for `tateless-function` is fine, you can keep it, since you have to follow project requirements. but others need to be removed.